docs/scene_walkthrough/performance_stress_test_walkthrough.md

```diff
@@ -18,4 +18,4 @@ This scene was created to test the performance of the overlay under different sy
 The Perfomance Stress Test Sample Scene is included in the `Extra Pack 2` sample pack.  
 
 > [!NOTE] 
-> Upon import, the `com.unity.postprocessing` package that this scene requires will be automatically installed.  You will still need to <a href="/readme.md#importing-the-samples">add it in the build settings</a>.  
\ No newline at end of file
+> Upon import, the `com.unity.postprocessing` package that this scene requires will be automatically installed.  You will still need to <a href="/README.md#importing-the-samples">add it in the build settings</a>.  
```

Comment:
This link is broken. 
![image](https://github.com/PlayEveryWare/eos_plugin_for_unity/assets/50968607/7b1f13bc-e40d-46e7-978b-11470890cc8b)

Reply:
This is an odd behavior with GFM - currently, if you have a link within a banner section, it is not altered to be relative to the repository, but relative to github.com proper :\

Temporarily resolved this by removing the link from being within a banner.